### PR TITLE
add onended event to sound component

### PR DESCRIPTION
**Pull request by meta-meta**

### CODE REVIEW

**Comment by meta-meta** [comment]
Travis is complaining about an unnecessary call() but I think it's the proper thing to do. Need to call the function with a this ref. Seems more unnecessary to say `sound.unended.bind(sound)()`

**Comment by ngokevin** [comment]
Should probably do the ```bind``` method since that is the intent, and just to please the standard linter.

File: src/components/sound.js [hunk 1]
```diff
@@ -104,6 +104,12 @@ module.exports.Component = registerComponent('sound', {
       });
     }
 
+    sound.source.onended = function () {
+      // we've "overridden" sound's onended binding so call it here
```
Comment (ngokevin) [inline src/components/sound.js]: If we use ```.bind```, does this comment supply us with any info?

**Comment by meta-meta** [comment]
maybe I'm crazy. I could swear `this` evalutated to `window` when not binding. object.fn.call(object) is indeed equivalent to object.fn()

**Comment by ngokevin** [comment]
Yeah, I was wondering why we couldn't just do ```sound.onEnded```

File: src/components/sound.js [hunk 2]
```diff
@@ -104,6 +104,12 @@ module.exports.Component = registerComponent('sound', {
...
+      sound.onEnded();
+      el.emit('onended');
```
Comment (ngokevin) [inline src/components/sound.js]: Let's emit ```sound-ended```

File: src/components/sound.js (hunk 1 above)
Comment (ngokevin) [inline src/components/sound.js]: Don't think we need this comment then

File: src/components/sound.js (hunk 2 above)
Comment (ngokevin) [inline src/components/sound.js]: Can also add this event to the sound docs https://github.com/aframevr/aframe/blob/dev/docs/components/sound.md#properties

Similar to https://github.com/aframevr/aframe/blob/dev/docs/components/material.md#events

**Comment by dmarcos** [comment]
There are conflicts. Needs rebase. We also moved development to master.  @meta-meta Would you mind to reopen the PR against master? Sorry for the inconvenience.

**Comment by dmarcos** [comment]
These changes landed on https://github.com/aframevr/aframe/pull/1106

**Pull request closed without merging**